From 55a291295799e13b9995f9f8ee3fac562ac91482 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Thu, 11 Jul 2019 16:53:27 -0700
Subject: [PATCH] joyent/node-sshpk-agent#19 update jsstyle submodule so make
 check works again Reviewed by: Cody Peter Mello <cody.mello@joyent.com>

---
 deps/javascriptlint | 2 +-
 deps/jsstyle        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/deps/javascriptlint b/deps/javascriptlint
index 040bf5e..ad52812 160000
--- a/deps/javascriptlint
+++ b/deps/javascriptlint
@@ -1 +1 @@
-Subproject commit 040bf5e429969ae42d976b570fd64c9e17fab20f
+Subproject commit ad52812e77bdfb1e90fb71a1201adb2b665a27e6
diff --git a/deps/jsstyle b/deps/jsstyle
index 194dde7..52dc973 160000
--- a/deps/jsstyle
+++ b/deps/jsstyle
@@ -1 +1 @@
-Subproject commit 194dde793f58b707b0416a106e1f7679795e50df
+Subproject commit 52dc973cf64da11834eca7cf46ebce8518e3ee88
-- 
2.21.0

